src/flamenco/runtime/fd_executor.c

```diff
@@ -1524,7 +1544,7 @@ fd_execute_txn( fd_exec_txn_ctx_t * txn_ctx ) {
         ret = fd_sysvar_instructions_update_current_instr_idx( txn_ctx, i );
         if( ret != FD_ACC_MGR_SUCCESS ) {
           FD_LOG_WARNING(( "sysvar instructions failed to update instruction index" ));
-          return ret;
+          goto ret;
```

Comment:
use a variable name as a label is very confusing

Reply:
good point: renamed as `exit`